packages/manager/apps/web/client/app/hosting/multisite/translations/Messages_fr_FR.json

```
@@ -8,5 +8,6 @@
   "hosting_multisite_statistics_period_week": "Semaine",
   "hosting_multisite_statistics_period_month": "Mois",
   "hosting_multisite_statistics_period_year": "Année",
-  "hosting_multisite_statistics_loading_error": "Une erreur est survenue lors de la récupération des statistiques"
+  "hosting_multisite_statistics_loading_error": "Une erreur est survenue lors de la récupération des statistiques",
```

We have as habit to display API message error.
I know it was here before, may be we can add it.

```suggestion
  "hosting_multisite_statistics_loading_error": "Une erreur est survenue lors de la récupération des statistiques : {{ error }}",
```

---

This was the existing, legacy message. He did not change it. If required, this change has to be included in a separate PR.